wmn-data.json

```diff
@@ -1028,6 +1028,15 @@
         "known" : ["doctypeme", "ocean.war"],
         "cat" : "coding"
        },
+      {
+        "name" : "CodeChef",
+        "uri_check" : "https://www.codechef.com/users/{account}",
+        "e_code" : 200,
+        "e_string" : "| CodeChef User Profile for ",
+        "m_code" : 302,

```

Comment:
Looks like the `m_string` (yes, I know the CONTRIBUTING said it was optional....that is fixed) is missing. Consider using something like `username specified does not exist`  for the `m_string`.